### ISSUE TITLE
[BUG] Updated to latest version, mass list of users will be deleted?

### ISSUE THREAD

**Issue by BluesFanUK** [AAndyProgram/SCrawler#204]
2024-07-07 09:28:55: The user [Reddit: Frostedcupcake92]  was not found. It will be removed from SCrawler on 2024-07-07 09:28:55.

I've literally got a list in the log of a hundred+ users the program has decided they no longer exists. Is this a new setting? Most of the users DO exist.

**Comment by AAndyProgram** [comment]
Have you changed the user's location? Drive letter?
Nothing has changed about this. SCrawler shows you this notification when it cannot find a user in its location.

**Comment by BluesFanUK** [comment]
No changes to the location, my PC has been off for a few days, I updated it today as it popped up with an update, once it reopened I got that message, it's removed them from the SCrawler program, and it won't allow me to add them back in as a list (just says 0 imported, they already exist etc), I can add them one by one though.

**Comment by AAndyProgram** [comment]
Do you update SCrawler every time I publish a new release? Have you updated SCrawler after v2024.2.25.0? Because major changes in the settings files were made in v2024.4.10.0.

**Comment by BluesFanUK** [comment]
Yep, my PC is usually on 24/7, so this is running all the time, if it flashes up I close it and install it. This week it was off due the weather for 3 days, I turned it on this morning, opened SCrawler, clicked Update, closed the program to complete the upgrade, then that flashed up.

In any case i'm not sure why it's trying to auto-remove profiles anyway, if it can't find something i'd expect the profile to turn red. I can't add 100+ profiles back in bulk withouit doing it 1 by 1.

**Comment by AAndyProgram** [comment]
**A profile is marked in red only it is not found on the site.**
Please show me the files `Settings.xml` and `Responser_Reddit_Settings.xml` from SCrawler settings folder and `User_Reddit_Frostedcupcake92.xml` from this user's settings folder.

**Comment by AAndyProgram** [comment]
Since I haven't received your response, I consider this issue resolved.
